<launch>    
    <node pkg="camera_info" type="camera_info_node" name="camera_info_node"></node>
    
    <node ns="stereo" pkg="stereo_image_proc" type="stereo_image_proc" name="stereo_image_proc" output="screen">
        
        <remap from="/stereo/left/image_raw" to="/cam0/image_raw"/>
        <remap from="/stereo/left/camera_info" to="/stereo_camera/left/camera_info"/>

        <remap from="/stereo/right/image_raw" to="/cam1/image_raw"/>
        <remap from="/stereo/right/camera_info" to="/stereo_camera/right/camera_info"/>
        
    </node>

    <!-- <node pkg="rqt_reconfigure" type="rqt_reconfigure" name="rqt_reconfigure"/> -->
</launch>